index.js

```diff
@@ -10,6 +10,13 @@ module.exports = {
     this._super.included.apply(this, arguments);
 
     this.addonOptions = app.options[this.name] || {};
+
+    if (this.addonOptions.plugins) {
+      this.addonOptions.plugins.forEach((plugin) => {
+        this.import(`node_modules/lazysizes/plugins/${plugin}/ls.${plugin}.js`);
```

Comment:
I see some issues here:
* the module resolution is too static here, we cannot assume the `lazysizes` npm package is in the `node_modules` folder (I guess this is relative to the addon?). it could be in another `node_modules` up the folder tree. You should use `let lazySizesPath = path.dirname(require.resolve('lazysizes'));` to let node.js resolve the package for you. See `treeForVendor`
* the plugins code will probably break in FastBoot, as it uses browser APIs. So all the plugin code should be wrapped in a `if (typeof FastBoot === 'undefined')` check. This is already done for the main library, and should be done for all used plugins as well. See `treeForVendor`. Instead of the manual wrapping, https://github.com/kratiahuja/fastboot-transform should probably be used!